app.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name": "readerapi",
+    "title": "Reader Api",
+    "description": "Reader Api",
+    "version": "0.0.1",
+    "year": "2017",
+    "language": "us",
+    "cookieId": "dc7161be3dbf2250c8954e560cc35061",
+    "author": {
+    	"name": "Gorillab",
+    	"email": "hello@gorillab.co",
+    	"url": "http://gorillab.co",
+    	"location": "Saigon"
+    }
+}
```

Comment:
use `dotenv` to store environment variables instead https://github.com/motdotla/dotenv